test/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ macro(onedpl_add_test test_source_file)
         DEPENDS ${_test_name}
         COMMENT "Build and run test ${_test_name}")
 
+    target_compile_options(${_test_name} PRIVATE "-Wall" PRIVATE "-Wformat-security" PRIVATE "-Werror=format-security" PRIVATE "-Wno-unused-lambda-capture" PRIVATE "-Wextra")
```

Comment:
1. What about cl-like compilers on Windows?
2. You don't need to put `PRIVATE` after each options you can just do like this: `target_compile_options(${_test_name} PRIVATE -Wall -Wformat-security ...)`

Reply:
1. Keys are currently added for compilers on linux to make catching for warnings available. In future updates keys will be added on Linux and Windows separately.